framework/transport/ieee1905_transport/include/mapf/transport/ieee1905_transport.h

```diff
@@ -161,10 +162,13 @@ class Ieee1905Transport {
         // implement Compare for the std::map template
         bool operator()(const DeDuplicationKey &lhs, const DeDuplicationKey &rhs) const
         {
-            // compare based on src address first then messageId, then fragmentId
+            // compare based on src and dst address first then messageId, then fragmentId
             int srccmp = memcmp(lhs.src, rhs.src, ETH_ALEN);
             if (srccmp)
                 return srccmp < 0;
+            int dstcmp = memcmp(lhs.dst, rhs.dst, ETH_ALEN);
+            if (dstcmp)
+                return dstcmp < 0;
             else if (lhs.messageType != rhs.messageType)
```

Comment:
(nitpick)
no need for the `else if` since `return` is performed if the previous condition is met